include/libgit4cpp/Repository.h

```diff
@@ -63,6 +63,8 @@ inline std::ostream& operator<<(std::ostream& stream, const RepoState& repostate
     return stream;
 }
 
+enum class BranchType {ALL = 0, LOCAL =1, REMOTE=2};
```

Comment:
According to the style guide, enumerators should be in snake_case:
```suggestion
enum class BranchType {all = 0, local =1, remote=2};
```
ALL_CAPS is reserved for macros.